Practise 18.11.20

YaroslavKomarov opened the pull request.

YaroslavKomarov committed:
Added .gitignore.

YaroslavKomarov committed:
practise start

solodyankina commented on Practise_18.11.20/Practise_18.11.20/Practise_18.11.20.cpp:
```diff
@@ -0,0 +1,33 @@
+#include "Maze.h"
+using namespace std;
+
+void ChangeMaze(Maze& maze, const int& width, const int& height) {
```
без доп методов

solodyankina reviewed (commented):
исключить лишние файлы, оставить только cpp h

YaroslavKomarov committed:
Removed additional methods;
Fixed the .gitignore file.

YaroslavKomarov committed:
Deleted .vcxproj file;
Deleted .sln file;
Deleted .vcxproj.filters file.

YaroslavKomarov committed:
Removed the additional GetChar method

YaroslavKomarov committed:
Added class MTreeNode;
Improved the hasConnection method in Maze.cpp;
A program has been written to display a new maze and tree.

solodyankina commented on Practise_18.11.20/Practise_18.11.20/Practise_18.11.20.cpp:
```diff
@@ -1,12 +1,64 @@
 #include "Maze.h"
 using namespace std;
 
+void BuildWebStepMaze(Maze& maze, const int& maze_size) {
```
нужно ли выделять этот метод?

YaroslavKomarov replied:
Я могу поместить этот метод внутри main, если это необходимо, но ведь ТЗ не запрещает вынести заполнение лабиринта в отдельный метод

solodyankina replied:
а зачем? если для этого предусмотрен метод

YaroslavKomarov committed:
В заголовочных вайлах теперь только объявления пременных и функций.
